tests/UI/pages/BO/catalog/productsV2/add/virtualProductTab.ts

```diff
@@ -0,0 +1,68 @@
+// Import page
+import BOBasePage from '@pages/BO/BObasePage';
+import ProductFaker from '@data/faker/product';
+
+import type {Page} from 'playwright';
+
+/**
+ * Virtual product tab on new product V2 page, contains functions that can be used on the page
+ * @class
+ * @extends BOBasePage
+ */
+export default class VirtualProductTab extends BOBasePage {
```

Comment:
```suggestion
class VirtualProductTab extends BOBasePage {
```
No need to export class unless you extending it somewhere

Reply:
Done thx :)